docs/index.rst

```diff
@@ -5,66 +5,85 @@ Tahoe-LAFS
 .. Please view a nicely formatted version of this documentation at
    http://tahoe-lafs.readthedocs.io/en/latest/
 
-Contents:
+Tahoe-LAFS is a Free and Open decentralized storage system.
+It distributes your data across multiple servers.
+Even if some of the servers fail or are taken over by an attacker,
+the entire file store continues to function correctly,
+preserving your privacy and security.
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
+   :caption: Introduction
 
```

Comment:
I would recommend renaming this to "Getting Started with Tahoe-LAFS" and include installation, and basic configuration articles". The "Using Tahoe-LAFS" section can cater to parallel stuff like some of the stuff you have mentioned (anonymizing network,CLI commands, etc).

Reply:
I'm not sure about this.  Introduction is, well, an introduction, and the rest (installing, building, configuring, running, other usage notes) should be under "Using Tahoe-LAFS".

Please take a look at [this build of the docs](https://tahoe-lafs--1030.org.readthedocs.build/en/1030/).  Would you still re-arrange it that way?

Reply:
Yes, thanks for the new build. I would definitely re-arrange it that way. I am not a big fan of having an Introduction section. Feels very retro to me, like we're documenting for print media. A lot of cloud tech companies are still bound to this ideology. See "[The grand theory of Documentation](https://documentation.divio.com/)". I am not a big fan of this and believe this does not work with writing for the internet. Instead, we can cut the clutter and just have two types of articles "About Tech1/Feature1". How to do "Tech1/Feature1".

Sorry about the tangent, but that is the reason I don't want to call it Introduction is because for people to read introductions they have to be motivated/excited about what they're reading about. 95% of the time the user is at help because they couldn't make stuff work or just want to learn **how to do specific things**. And that's the reason I want all titles to (somewhat) be verbs. i.e.,

- About Tahoe-LAFS > Welcome to Tahoe-LAFS
- Getting Started > Introduction
- Installing Tahoe-LAFS > Installation Guide
- Running Tahoe-LAFS > How to Run Tahoe-LAFS

Of course, there are going to be some theoretical articles, but that's why we will have "About ..." articles; where we will actively drive the user to a "How-to" article.

About Nodes will (potentially) redirect users to:
- Configuring a Tahoe-LAFS node
- Running Tahoe-LAFS

I hope that makes sense.

Reply:
@YashNRam13 Made a change in f39d765.  The other changes you describe will have to be separate pull requests, right?

Reply:
I'm 100% in agreement with the structure:
> About Tahoe-LAFS > Welcome to Tahoe-LAFS
> Getting Started > Introduction
> Installing Tahoe-LAFS > Installation Guide
> Running Tahoe-LAFS > How to Run Tahoe-LAFS

Reply:
I do like the new structure!